packages/haxe/haxe.3.4.7/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: [
+  "Haxe Foundation <contact@haxe.org>" "Andy Li <andy@onthewings.net>"
+]
+authors: "Haxe Foundation <contact@haxe.org>"
+homepage: "https://haxe.org/"
+bug-reports: "https://github.com/HaxeFoundation/haxe/issues"
+license: ["GPL2+" "MIT"]
+dev-repo: "https://github.com/HaxeFoundation/haxe.git"
+build: [
+  ["git" "submodule" "update" "--init" "--recursive"]
+  ["env" "OCAMLPARAM=safe-string=0,_" make]
+]
+install: [
+  ["cp" "haxe" "haxelib" "%{bin}%"]
+  ["mkdir" "-p" "%{share}%/haxe"]
+  ["cp" "-r" "std" "%{share}%/haxe/std"]
+]
+remove: [
+  ["rm" "%{bin}%/haxe" "%{bin}%/haxelib"]
+  ["rm" "-r" "%{share}%/haxe"]
+]
+depends: [
+  "ocamlfind" {build}
+  "camlp4" {build}
+  "conf-neko"
+]
+available: [ocaml-version > "4.02"]
```

Comment:
Do you mean > 4.02.0 here or >= 4.03.0 ?
In the former case 4.02.3 is accepted for example.